bip-update-process.md

```diff
@@ -0,0 +1,706 @@
+the novelty of an idea can be tested by posting about it to the [Bitcoin Development Mailing
+List](https://groups.google.com/g/bitcoindev). Prior correspondence can be found in the [mailing list
+archive](https://gnusha.org/pi/bitcoindev/).
+
+Vetting an idea publicly before investing the time to describe the idea formally is meant to save both the authors and
+the broader community time. Not only may someone point out relevant discussion topics that were missed in the authors’
+research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
+tests whether it is of interest to more people besides the authors. After establishing that the idea may be of interest
+to the Bitcoin community, the authors should work on drafting a BIP.
+
+As a first sketch of the proposal is taking shape, the authors should present it to the [Bitcoin Development Mailing
+List](https://groups.google.com/g/bitcoindev). This gives the authors a chance to collect initial feedback and address
+fundamental concerns. If the authors wish to work in public on the proposal at this stage, it is recommended that they
+open a pull request against one of their forks of the BIP repository instead of the main BIP repository.
+
+It is recommended that complicated proposals be split into separate BIPs that each focus on a specific component of the
+overall proposal.
+
+### Progression through BIP Statuses
+
+The following sections refer to BIP Status Field values. The BIP Status Field is defined in the Header Preamble
+specification above.
+
+#### Draft
+
+After fleshing out the proposal further and ensuring that it is of high quality and properly formatted, the authors
+should open a pull request to the [BIP repository](https://github.com/bitcoin/bips). The document must adhere to the
+formatting requirements specified above and should be provided as a file named with a working title of the form
+"bip-title.[md|mediawiki]". The authors must not self-assign a number to their proposal.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed beyond the
+ideation phase, e.g., by generating substantial public discussion and commentary from diverse contributors, by
+independent Bitcoin projects working on adopting the proposal, or by the authors working for an extended period toward
+improving the proposal based on community feedback, will be assigned a number by a BIP editor. The BIP editors should
+delay number assignment when they perceive a proposal being met with lack of interest: number assignment facilitates the
+distributed discussion of ideas, but before a proposal garners some interest in the Bitcoin community, there is no need
+to refer to it by a number.
+
+Proposals are also not ready for number assignment if they duplicate efforts, disregard formatting rules, are too
```

Comment:
A mention of or reference to the scope of BIPs described lines 45-47 would be helpful in this sentence.

Reply:
If something is out of scope, it’s offtopic, so it seems unnecessary to additionally mention that it’s not eligible for number assignment.